# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ilmbase/gcc43.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ilmbase-1.0.2/Imath/ImathMatrix.h.vanilla	2010-07-28 17:15:50.000000000 +0200
+++ ilmbase-1.0.2/Imath/ImathMatrix.h	2010-07-28 17:16:03.000000000 +0200
@@ -49,6 +49,7 @@
 #include "ImathVec.h"
 #include "ImathShear.h"
 
+#include <string.h>
 #include <iostream>
 #include <iomanip>
 
